pkg/crc/machine/machine.go

```diff
@@ -801,7 +801,7 @@ func configProxyForCluster(ocConfig oc.Config, sshRunner *crcssh.Runner, sd *sys
 
 	logging.Info("Adding proxy configuration to the cluster ...")
 	proxy.AddNoProxy(instanceIP)
-	if err := cluster.AddProxyConfigToCluster(ocConfig, proxy); err != nil {
+	if err := cluster.AddProxyConfigToCluster(sshRunner, ocConfig, proxy); err != nil {
```

Comment:
`sshRunner` and `ocConfig` are slightly redundant, wondering if we should create the `ocConfig` 'on demand' in the cluster package, or have a way to get the ssh runner from the `ocConfig`? Thinking out loud without spending much time on the existing code, maybe none of these suggestions make sense :)

Reply:
Not right now because the interface we use have only 2 methods so we can support ocConfig from the host or from the VM, we can change it if we are moving away with host oc config but that would be a different refactor.